### ISSUE TITLE
Add option to also remove uploaded files (if upload is enabled) with the storage cleaner

### ISSUE THREAD

**Issue by TehPeGaSuS** [thelounge/thelounge#4846]
### Feature Description
Now that storage cleaning was added with #4799, it'd be nice if it also deleted the uploaded files (if upload is enabled), since those are the ones that take up most of the disk space.

**Comment by Yogui79** [comment]
hello, 
I also interested to this feature and plan to add it
my idea is to add a  key in the fileUpload object it would look like this : 

```
  fileUpload: {
		enable: false,
		maxFileSize: 10240,
		maxAge : null,
		baseUrl: null,
	},

```
the possible value for the max age could be  :  `null` (disabled), `"X minutes"`,  `"X hours"` or `"X days"` 

what do thin about that

**Comment by brunnre8** [comment]
s/null/undefined/g
And the max age should be days. We neither need to implement a funny parser nor do we need *that* granularity.
Uploads that go away a second later are not useful. The whole point is giving people some time to look at it

**Comment by TehPeGaSuS** [comment]
I agree. anything less than 1d isn't very useful, IMHO. and anything over than 7d are just lingering.

**Comment by Yogui79** [comment]
honestly I put minute only to complete the range. Actualy  I delete my file older that 5 hours with a cron but you right 1 day is a good range.

whats about delete at midnight file older that the define days value ?

**Comment by brunnre8** [comment]
no need to do it at a specific time, all you care about is the delta.
Doesn't really matter if that's at midnight or at 4 o'clock

**Comment by SawkeeReemo** [comment]
Is there any documentation on how to set this up? I have several gigs of logs that need to be cleared out, but I'm not finding any info in the wiki or elsewhere (yet).
